src/main/resources/templates/courses/assignments/group-pulls.ftl

```diff
@@ -37,7 +37,7 @@
                               </div>
                               <div class="pull-right"><i class="glyphicon glyphicon-comment"></i> ${pullRequest.comments?size} </div>
                             </span>
-                            <div class="comment">Pull Request #${pullRequest.issueId}: ${pullRequest.branchName}</div>
+                            <div class="comment">${pullRequest.title}</div>
```

Comment:
The line https://github.com/devhub-tud/devhub/pull/384/files#diff-e3b9c5a72b297c49ed8754b7d71cda8aR74 should be updated as well.